File: Senior Yr/cs380/Assignments/Assignment1/src/SlidingBrickPuzzle/Position.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package SlidingBrickPuzzle;

/**
 *
 * @author sunnypatel
 */
public class Position {
    int x;
    int y;
    
    public Position(){
        x = 0;
        y = 0;
    }
    
    public Position(int x, int y){
        this.x = x;
        this.y = y;
    }
    
    public int getX(){
        return this.x;
    }
    
    public void setX(int x){
        this.x = x;
    }
    
    public int getY(){
        return this.y;
        
    }
    
    public void setY(int y){
        this.y = y;
    }
    
}

File: Senior Yr/Fall/cs457/PA2-tests/src/Position.cpp
/* 
 * File:   Position.cpp
 * Author: sunnypatel
 * 
 * Created on November 16, 2013, 4:23 PM
 */

#include <stdlib.h>

#include "Position.h"

Position::Position() {
    this->x = 0;
    this->y = 0;
    
}

Position::Position(const Position& orig) {
}

Position::~Position() {
}

/**
 * check if position is out of bounds
 * @param state
 * @return 
 */
//bool outOfBounds(Puzzle state){
  //  return false;
//}

/**
 * Returns the X coordinate
 * @return 
 */
int Position::getX(){
    return this->x;
}
/**
 * Return the Y coordinate
 * @return 
 */
int Position::getY(){
    return this->y;
}

void Position::setX(int x){
    this->x = x;
}

void Position::setY(int y){
    this->y = y;
}
